sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -202,6 +202,27 @@ class SpinTable(Base):
     ingestion_date = Column(DateTime(timezone=True))
 
 
+class PointingTable(Base):
+    """Pointing table."""
+
+    __tablename__ = "pointing_table"
+    pointing_id = Column(Integer, nullable=False, primary_key=True, unique=True)
+    pointing_start_utc = Column(DateTime(timezone=True), nullable=False)
+    pointing_end_utc = Column(DateTime(timezone=True))
+    repoint_start_utc = Column(DateTime(timezone=True))
+    repoint_end_utc = Column(DateTime(timezone=True))
+
+
+class RepointTable(Base):
```

Comment:
Would this be easier to distinguish if we added in a "Files" qualifier because it is really just a file tracker.
```suggestion
class RepointFiles(Base):
```

Reply:
I think I want to keep this same as other to not confuse with other functions from imap-data-access.

Reply:
But the other similar tables here are called `ScienceFiles`, `AncillaryFiles`, and `SPICEFiles`. `SpinTable` is the only one with Table in the name, so I think this would be consistent with what I was expecting on the ScienceFiles and how that is stored?

Reply:
done